build/params_2k.go

```diff
@@ -36,7 +36,7 @@ func init() {
 	policy.SetConsensusMinerMinPower(abi.NewStoragePower(2048))
 	policy.SetMinVerifiedDealSize(abi.NewStoragePower(256))
 
-	BuildType |= Build2k
+	BuildType = Build2k
```

Comment:
This was like that because in debug mode 2k params (this file) also execute, which may result in a wrong version in debug mode